### PR TITLE
[TT-1388] fixing uptime check err msg

### PR DESCRIPTION
**Pull request by tbuchaillot**
## Description

Fixing uptime tests check error message. It will not show an error, now it shows a warning and only when error !=ErrPoolNotRunning

## Related Issue

https://tyktech.atlassian.net/browse/TT-1388
## Motivation and Context

https://tyktech.atlassian.net/browse/TT-1388
## How This Has Been Tested

Tests details on https://tyktech.atlassian.net/browse/TT-1388
## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If pulling from your own
      fork, don't request your `master`!
- [ ] Make sure you are making a pull request against the **`master` branch** (left side). Also, you should start
      *your branch* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
  - [ ] If you've changed APIs, describe what needs to be updated in the documentation.
  - [ ] If new config option added, ensure that it can be set via ENV variable
- [ ] I have updated the documentation accordingly.
- [ ] Modules and vendor dependencies have been updated; run `go mod tidy && go mod vendor`
- [ ] When updating library version must provide reason/explanation for this update.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Check your code additions will not fail linting checks:
  - [ ] `go fmt -s`
  - [ ] `go vet`

### CODE REVIEW

**Review by sredxny — APPROVED** [review]
LGTM! 
As a note, this is another situation that we could cover with MDCB testing framework

**Review by matiasinsaurralde — APPROVED** [review]

File: gateway/host_checker.go [hunk 1]
```diff
@@ -136,8 +136,8 @@ func (h *HostUptimeChecker) execCheck() {
 	h.resetListMu.Unlock()
 	for _, host := range h.HostList {
 		_, err := h.pool.SendWork(host)
-		if err != nil {
-			log.Errorf("[HOST CHECKER] could not send work, error: %v", err)
+		if err != nil && err != tunny.ErrPoolNotRunning {
+			log.Warn("[HOST CHECKER] could not send work, error: %v", err)
```
Comment (buger) [inline gateway/host_checker.go:140]: This error in check is totally valid. It should be Warnf function.

Comment (tbuchaillot) [reply gateway/host_checker.go:140]: @buger why should we Warn the msg if the pool is not running?

**Comment by buger** [comment]
@tbuchaillot needs manual merge to 2.9

**Comment by tbuchaillot** [comment]
Merged on `release-2.9` on this PR https://github.com/TykTechnologies/tyk/pull/3504

**Pull request merged**